elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -238,21 +197,28 @@ public Object getRelation(
                 @SuppressWarnings("unchecked")
                 Class<?> relationClass = dictionary.getParameterizedType(entity, relationName);
 
+                RelationshipImpl relationship = new RelationshipImpl();
```

Comment:
Can we just give this an `@AllArgsConstructor`? These things should either be fully constructed or not at all, correct?

We could also then make it immutable.